packages/block-library/src/paragraph/edit.native.js

```diff
@@ -95,13 +116,16 @@ class ParagraphEdit extends Component {
 		} = attributes;
 
 		return (
-			<View>
+			<View
+				accessible={ ! this.state.isSelected }
```

Comment:
Here we should probably use `isSelected` from props. In that way it will be properly set to `false` when the block is selected by any method (not just from focusing RichText directly), and ScreenReader won't read this label after double tap to select.